javascript/sails/config/env/development.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
there is no **production.js** ?

Reply:
You're right too, I rename this file in production.js

Reply:
I don't know how you start the process app, but you'll have to set `NODE_ENV` to `production` to make sails consider the `production.js` file.

Reply:
yep, done in `Dockerfile`